pkg/requestutils/request.go

```diff
@@ -61,6 +62,9 @@ func (r *requests) MakeAPIRequest(
 		}
 		defer resp.Body.Close()
 		statusCode = resp.StatusCode
+		if retryableStatusCode(statusCode) {
```

Comment:
why not simply?
```suggestion
		if 500 <= statusCode && statusCode < 600 {
```

Reply:
Because for some status code we may not  want to retry ex - `StatusNotImplemented`

Reply:
Noted and changed